lib/GLPI/Agent/Task/Inventory/Linux/AntiVirus/EEA.pm

```diff
@@ -0,0 +1,89 @@
+package GLPI::Agent::Task::Inventory::Linux::AntiVirus::EEA;
+
+use strict;
+use warnings;
+
+use parent 'GLPI::Agent::Task::Inventory::Module';
+
+use Time::Local;
```

Comment:
I noticed a warning in the Time::Local man page:
>       Warning: The year value interpretation that these functions and their nocheck variants use will almost certainly lead to bugs in your code, if not now, then in the future. You are strongly discouraged from using these in new code, and you should convert old code to using
>       either the *_posix or *_modern functions if possible.

For the same purpose I discover you can use the mktime posix function, so here is use update and below the code one:

```suggestion
use POSIX qw(mktime);
```